docs/sources/developers/http_api/access_control.md

```diff
@@ -566,6 +566,7 @@ Lists the roles that have been directly assigned to a given user. The list does
 Query Parameters:
 
 - `includeHidden`: Optional. Set to `true` to include roles that are `hidden`.
+- `includeMapped`: Optional. Set to `true` to include roles that have been mapped through the role attribute sync feature.
```

Comment:
~role~ *group* attribute sync

sorry... I was trying to figure out how to suggest a change directly and it's escaping me.

Reply:
Ah yes, you're right. Just updated the wording.